Formula/kubesess.rb

```diff
@@ -19,6 +19,25 @@ class Kubesess < Formula
 
   def install
     system "cargo", "install", *std_cargo_args
+    bash_completion.install "./scripts/sh/completion.sh"
+    zsh_function.install "./scripts/sh/kubesess.sh"
+
+    fish_completion.install "./scripts/fish/completions/kc.fish"
+    fish_completion.install "./scripts/fish/completions/kn.fish"
+    fish_completion.install "./scripts/fish/completions/knd.fish"
+    fish_completion.install "./scripts/fish/completions/kcd.fish"
+
+    fish_function.install "./scripts/fish/functions/kc.fish"
+    fish_function.install "./scripts/fish/functions/kcd.fish"
+    fish_function.install "./scripts/fish/functions/kn.fish"
+    fish_function.install "./scripts/fish/functions/knd.fish"
+  end
+
+  def caveats
+    <<~EOS
+      To use kubesess it is required to enable completions.
+         Read more here: https://docs.brew.sh/Shell-Completion
+    EOS
```

Comment:
```suggestion
```
This isn't specific to Homebrew.

Reply:
Removed this.

Reply:
I will add the specifics in the installation instructions instead.